sql/core/src/main/scala/org/apache/spark/sql/execution/joins/BroadcastHashJoinExec.scala

```diff
@@ -46,7 +46,7 @@ case class BroadcastHashJoinExec(
     left: SparkPlan,
     right: SparkPlan,
     isNullAwareAntiJoin: Boolean = false)
-  extends HashJoin with CodegenSupport {
+  extends HashJoin {
```

Comment:
This is just a minor fix packed in this PR. `HashJoin` already extends `JoinCodegenSupport`. So we don't need `CodegenSupport` here.

Reply:
If you don't mind, could you make a separate PR? You can use `[MINOR]` tag instead of creating a new JIRA.

Reply:
@dongjoon-hyun - sure, will do it.